h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -1067,6 +1067,10 @@ public final BufferedString atStr( BufferedString bStr, long i ) {
     } else return chunkForRow(i).atStr_abs(bStr, i);
   }
 
+  public String stringAt(int i) {
```

Comment:
this method is not used and the parameter should be long since it is defined on a Vec